Week 1 Recipe Page/styles.css

```diff
@@ -11,11 +11,12 @@ body{
 }
 .header {
     display: flex;
```

Comment:
To help make code review easier, I typically add a screenshot to my code review.

This looks like it make sense !